plugins/evaluation-plugin/src/com/intellij/cce/actions/CompletionEvaluationStarter.kt

```diff
@@ -36,8 +38,11 @@ internal class CompletionEvaluationStarter : ApplicationStarter {
       .main(args.toList().subList(1, args.size))
 
   abstract class EvaluationCommand(name: String, help: String) : CliktCommand(name = name, help = help) {
-    protected fun loadConfig(configPath: Path) = try {
-      ConfigFactory.load(configPath)
+
+    protected val featureName by argument(name = "Feature name").default("rename")
```

Comment:
let's rename command name to `evaluate` instead of `evaluate-completion` (in the beginning of class)